app/assets/javascripts/katello/containers/container.js

```diff
@@ -55,6 +55,7 @@ KT.container = (function(){
     getTag = function() {
         return $('#tag_id').val();
     },
+    // TODO: This only returns Pulp 2 proxies, which no longer exist
```

Comment:
I can't judge if this whole page is now redundant (was it the Pulp 2 container feature?) or if it needs to be updated to Pulp 3 proxies.

Given it mentions the `docker` content type, I have a feeling it's Pulp 2.

Reply:
https://github.com/Katello/katello/pull/11078